common/api/ecschema-editing.api.md

```diff
@@ -751,7 +751,7 @@ export enum DiagnosticType {
 }
 
 // @beta (undocumented)
-export function diagnosticTypeToString(type: DiagnosticType): "CustomAttributeContainer" | "None" | "Property" | "RelationshipConstraint" | "Schema" | "SchemaItem";
```

Comment:
That is often an indication, that your local builds (aside of ecschema-editing) is out of date. Probably do a TMR, but to have extract-api check pass, the strings always need to be in alphabetical order